app/helpers/ems_infra_helper/textual_summary.rb

```diff
@@ -41,6 +41,12 @@ def textual_group_topology
   # Items
   #
 
+  def textual_region
+    return nil if @record.region_id.blank?
+    label_val = _('Region')
+    {:label => label_val, :value => @record.region_id}
```

Comment:
Perhaps
```ruby
{:label => _('Region'), :value => @record.region_id}
```
would look a tiny bit better. Plus rubocop complains that there should be a blank line after that `return nil` line.

Reply:
Agreed. Fixed :)